design-proposals/instancetype.kubevirt.io/extend-preferSpread.md

```diff
@@ -0,0 +1,325 @@
+# Overview
+
+The `preferSpread` `preferredCPUTopology` was introduced with KubeVirt v1.1 and allows for vCPUs provided by an instance type to be spread across guest visible sockets and cores using a configurable ratio that defaults to 2.
```

Comment:
is preferSpread a field name or value?
Same for preferredCPUTopology?

Maybe reword/reformat to make it clearer, i.e.:

> The `preferredCPUTopology: preferSpread` was introduced …

Reply:
`preferredCPUTopology` is the field and also associated string type but ACK I'll reword this.